Geveltuin

saschavanvliet opened the pull request:
Hoi Kerem en Nayome,

Het is mij gelukt om de geveltuin te stylen en dynamische inhoud in te laden. Ik zal even uitleggen en laten zien wat ik heb gedaan. 

Ik heb allereerst nauwkeurig gekeken naar de [Figma](https://www.figma.com/design/MhP0yeVhwLGq9ZtPltw1z9/Bieb-in-Bloei?node-id=3-15&p=f&t=LLO8Xvr3m23FtVh7-0) schetsen:

<img width="261" alt="image" src="https://github.com/user-attachments/assets/3ac7e3a4-25b4-4c88-b1dd-7a88ee2442ab" />

Deze heb ik vervolgens zo accuraat mogelijk geprobeerd na te maken.

https://github.com/user-attachments/assets/e016cdbc-790f-4e0f-8625-3e3b7ef08caf

Helaas is het mij niet gelukt om de achtergrond image voor de header goed te positioneren op de achtergrond. Ik heb daarom even een ander plaatje uit de assets als tussenplaatje gedaan.

Mijn vraag aan jullie is:
- Hoe fix ik dat het plaatje goed op de achtergrond komt? Hij is goed ingeladen, alleen het positioneren lukt niet..
- Zien jullie nog styling-wise fouten die ik moet veranderen?

Note to @Nayomekaia: ik heb ruimte overgelaten om de button erin te zetten! Dit moet namelijk onder de tekst:

<img width="1032" alt="image" src="https://github.com/user-attachments/assets/477b1346-67e4-41bd-ab97-0895b4eca7eb" />

saschavanvliet committed:
#7 Data ingeladen

- Serverdata meer gestructureerd door variabelen aan te maken
- Navigatie aangepast

saschavanvliet committed:
#7 Link naar goede pagina

saschavanvliet committed:
#11 Styles gefixt op de pagina

saschavanvliet committed:
#7 Iets meer styling opgelost

- Kleuren text in juiste variabelen
- H1 toegevoegd

saschavanvliet committed:
#7 Geveltuin pagina afgemaakt

- Alle inhoud erin gezet
- Responsive gemaakt met flex
- Carousel met images toegevoegd

saschavanvliet committed:
#7 Begin lazy load - performance

Nayomekaia commented:
Hi Sascha,

Ik weet niet precies welk plaatje je bedoelt zou je dat wat specifieker kunnen toelichten? In de video zie ik namelijk niets opvallends aan de achtergrond.

Verder ziet de CSS-code er netjes uit. Wel viel het me op dat het design afwijkt van het ontwerp dat we eerder samen hebben gemaakt. Er worden andere kleuren gebruikt, waardoor het visueel minder samenhangt. Ons originele design vond ik persoonlijk sterker, juist omdat het meer één geheel vormt.

<img width="300" alt="Scherm­afbeelding 2025-05-26 om 23 15 33" src="https://github.com/user-attachments/assets/24ba5e64-213a-4613-879c-040732599c43" />


<img width="300" alt="Scherm­afbeelding 2025-05-26 om 23 26 54" src="https://github.com/user-attachments/assets/5cb30a09-a681-4402-bc11-61642ca7073a" />


Ook merkte ik op dat de introductietekst vetgedrukt is, terwijl dit volgens het ontwerp regular hoort te zijn. Let op dat je de gemaakte designs aanhoud kwa kleuren vooral. 

Laat me weten als je hulp nodig hebt met het aanpassen!

saschavanvliet commented:
Hi @Nayomekaia, 

Dankjewel voor je feedback. Ik bedoel het plaatje dat achter de geveltuin tekst hoort te staan. Oftewel de header.

Ik bedoel deze hier:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/75cb7037-c291-47ac-a9b4-5f64b4762107" />

Nayomekaia commented:
Hii Sascha,
Ik heb zelf deze instellingen gebruikt voor mijn header achtergrond-image:

```
.header-title {
  background-image: url("/assets/stekjes-background.jpeg");
  background-size: cover;
  background-position: center;
  background-repeat: no-repeat;
  height: 90vh;

  & h1 {
    font-size: 4rem;
    display: flex;
    align-items: center;
    justify-content: center;
    height: 100%;
  }
}
```
background-size: cover zorgt ervoor dat de hele achtergrond bedekt wordt.
De positie is gecentreerd met background-position: center,
en no-repeat voorkomt dat de afbeelding zich herhaalt als deze niet groot genoeg is.

Ik vond zelf height: 100vh wat te lang, dus ik heb het aangepast naar 90vh voor een iets rustiger effect. 

Hopelijk heb je hier iets aan! Als je wilt, kunnen we er ook samen even naar kijken.

saschavanvliet committed:
#65 #7 Header gefixt

- Header gefixt (dynamische image)
- Positionering div met overlay gefixt (hij versprong steeds in 'inspecteren'.
- Nayomes feedback verwerkt

saschavanvliet commented:
@Nayomekaia, Ik heb de header gefixt! Zie je commit hierboven voor de code. Heel erg bedankt voor je feedback!

<img width="476" alt="image" src="https://github.com/user-attachments/assets/15ab395e-add7-406e-90f4-3421919777ba" />

Nayomekaia committed:
Merge branch 'main' into geveltuin
